src/Factories/FunFunFactory.php

```diff
@@ -81,7 +81,7 @@
 use WMDE\Fundraising\Frontend\Infrastructure\Repositories\LoggingSubscriptionRepository;
 use WMDE\Fundraising\Frontend\Infrastructure\TemplateBasedMailer;
 use WMDE\Fundraising\Frontend\Infrastructure\TokenGenerator;
-use WMDE\Fundraising\Frontend\Infrastructure\TwigCachePurger;
+use WMDE\Fundraising\Frontend\Infrastructure\AllOfTheCachePurger;
```

Comment:
Funny name, but, err, maybe something like `EntireCachePurger` or `CompleteCachePurger`? Not so sure, it sounds like the purger itself is entire or complete. Merging it anyway.